### ISSUE TITLE
Accessing info.cpu and info.iter

### ISSUE THREAD

**Issue by fungs** [matthias-k/cyipopt#17]
cyipopt is very useful, thanks for providing it.

The info dictionary returned by the solve method lacks information about the CPU runtime and the number of iterations needed. Of course, you can get this using the intermediate method or a callback, but other interfaces seem to have this. Any chance to include this as well?

**Comment by moorepants** [comment]
Seems like the matlab interface has this. I'm having trouble finding the source code for that package. It isn't clear to me that ipopt has an API for this info, but it is certainly displayed to the screen. If you can point me to the other implementations, I can see how they do it.

**Comment by moorepants** [comment]
I'm not seeing that the IPOPT C interface provides this.

**Comment by dance858** [comment]
We use cyipopt in the CVXPY extension to nonlinear programming. It's very easy to use. Thank you for this great piece of software.

We'd like to access the number of iterations from ipopt without defining a callback function. Has any progress been made on this issue since last time?

**Comment by moorepants** [comment]
I'm not aware of anyone working on this. I am happy to accept a pull request with the new feature.

**Comment by dance858** [comment]
> I'm not aware of anyone working on this. I am happy to accept a pull request with the new feature.

Thank you for such a quick response. Do you know roughly what we would need to change to expose this information?

**Comment by moorepants** [comment]
No, I wrote "I'm not seeing that the IPOPT C interface provides this." in 2021 above.
